api_app/connectors_manager/serializers.py

```diff
@@ -133,3 +133,33 @@ def get_verification(self, raw_instance):
             "error_message": final_err_msg,
             "missing_secrets": missing_secrets,
         }
+
+
+class ConnectorReportSerializer(serializers.ModelSerializer):
+    """
+    ConnectorReport model's serializer.
+    """
+
+    name = serializers.CharField(source="connector")
+    success = serializers.SerializerMethodField()
+    started_time_str = serializers.SerializerMethodField()
+
+    class Meta:
+        model = ConnectorReport
+        fields = (
+            "name",
+            "success",
+            "report",
+            "errors",
+            "process_time",
+            "started_time_str",
```

Comment:
To be changed to the new format when `AnalyzerReport` is up (since UI breaks now)